hs_tracking/signals.py

```diff
@@ -59,6 +62,24 @@ def capture_download(**kwargs):
     fields['resource_type'] = kwargs['resource'].resource_type
     fields['resource_guid'] = kwargs['resource'].short_id
 
+    # MakeDataCount fields
+    fields['tracking_visitor_id'] = session.visitor.id
+    fields['doi'] = kwargs['resource'].doi
+    fields['user_agent'] = kwargs['request'].headers.get('user-agent')
+
+    # MakeDataCount also requires the following metadata about the resource:
+    res_metadata = {
+        'title': kwargs['resource'].title,
+        'creators': " ;".join(str(name) for name in kwargs['resource'].metadata.creators.only('name').all()),
+        'is_replaced_by': kwargs['resource'].get_relation_version_res_url(RelationTypes.isReplacedBy) or None,
+        'is_version_of': kwargs['resource'].get_relation_version_res_url(RelationTypes.isVersionOf) or None,
+        'resource_url': current_site_url() + kwargs['resource'].absolute_url
+    }
+    if kwargs['resource'].raccess.published:
+        res_metadata['publisher'] = str(kwargs['resource'].metadata.publisher),
+        res_metadata['publish_date'] = kwargs['resource'].publish_date.strftime("%m/%d/%Y %H:%M:%S.%f")
```

Comment:
Is there a `publish_date` attribute for resource?

Reply:
Thanks for checking this! I had seen resource.publish_date:
```
>>> from hs_core.models import BaseResource
>>> BaseResource.objects.first().publish_date
datetime.datetime(2015, 10, 16, 17, 15, 31, 472440, tzinfo=<UTC>)
```
But looks like it comes from Mezzanine
https://github.com/stephenmcd/mezzanine/blob/v5.1.4/mezzanine/core/models.py#L253
so I will come up with a better way...